sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeClientOptions.cs

```diff
@@ -87,18 +88,22 @@ private void AddHeadersAndQueryParameters()
             Diagnostics.LoggedHeaderNames.Add("x-ms-owner");
             Diagnostics.LoggedHeaderNames.Add("x-ms-group");
             Diagnostics.LoggedHeaderNames.Add("x-ms-acl");
-            Diagnostics.LoggedHeaderNames.Add("continuation");
-            Diagnostics.LoggedHeaderNames.Add("maxResults");
-            Diagnostics.LoggedHeaderNames.Add("mode");
-            Diagnostics.LoggedHeaderNames.Add("recursive");
-            Diagnostics.LoggedHeaderNames.Add("action");
-            Diagnostics.LoggedHeaderNames.Add("upn");
-            Diagnostics.LoggedHeaderNames.Add("resource");
-            Diagnostics.LoggedHeaderNames.Add("directory");
-            Diagnostics.LoggedHeaderNames.Add("position");
-            Diagnostics.LoggedHeaderNames.Add("retainUncommittedData");
-            Diagnostics.LoggedHeaderNames.Add("close");
 
+            // Data Lake query parameters
+            Diagnostics.LoggedQueryParameters.Add("recursive");
+            Diagnostics.LoggedQueryParameters.Add("resource");
+            Diagnostics.LoggedQueryParameters.Add("action");
+            Diagnostics.LoggedQueryParameters.Add("position");
+            Diagnostics.LoggedQueryParameters.Add("retainUncommittedData");
+            Diagnostics.LoggedQueryParameters.Add("close");
+            Diagnostics.LoggedQueryParameters.Add("timeout");
```

Comment:
`timeout` wasn't in the removed diff above?

Reply:
Nope.